packages/liveblocks-core/src/notifications.ts

```diff
@@ -60,10 +62,14 @@ export function createNotificationsApi<M extends BaseMetadata>({
   }>;
 } {
   async function fetchJson<T>(
-    endpoint: string,
+    endpoint: URLSafeString,
```

Comment:
Really like this! Do you know if it's possible to enforce this argument to have a `/v2/c/` prefix on a type level?